virttest/utils_test/libvirt.py

```diff
@@ -3830,10 +3830,13 @@ def check_logfile(search_str, log_file, str_in_log=True,
     :param runner_on_target:  Remote runner
     :param ignore_status: True to return False on failure,
                           False to raise exception on failure
+    :param ignore_str: Ignore str
     :raise: test.fail when the result is not expected
     :return: True if check is successful, otherwise False
     """
     cmd = "grep -E '%s' %s" % (search_str, log_file)
+    if len(ignore_str) != 0:
```

Comment:
Do we need add ignore_statu=True as precondition?

Reply:
Hi, @chunfuwen 

> Do we need add ignore_statu=True as precondition?

No. I think the ignore_str and ignore_status should be irrelevant.
No matter ignore_status is True or False, we should also supports ignore_str to filter out unexpected strs.